configure.in: config PYTHON_SITE_PACKAGES

Config PYTHON_SITE_PACKAGES to instead of the incorrect test.

Upstream-Status: Inappropriate [WR-installer specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 61b2b4b..86ac99d 100644
--- a/configure.in
+++ b/configure.in
@@ -743,7 +743,7 @@ dnl
 
 PYTHON_VERSION=
 PYTHON_INCLUDES=
-PYTHON_SITE_PACKAGES=
+PYTHON_SITE_PACKAGES=$PYTHON_SITE_PACKAGES
 PYTHON_TESTS=
 pythondir=
 if test "$with_python" != "no" ; then
-- 
1.8.1.2

